server/src/uuid/model/entity/mod.rs

```diff
@@ -883,10 +883,16 @@ impl Entity {
                 FROM entity_revision r
                 JOIN uuid u_r ON r.id = u_r.id
                 JOIN entity e ON e.id = r.repository_id
+                JOIN type ON e.type_id = type.id
                 JOIN uuid u_e ON e.id = u_e.id
                 WHERE ( e.current_revision_id IS NULL OR r.id > e.current_revision_id )
                     AND u_r.trashed = 0
                     AND u_e.trashed = 0
+                    AND type.name NOT IN ("input-expression-equal-match-challenge",
```

Comment:
It is more intuitive to put the supported revisions (instead of a negative statement). So
```sql
AND type.name IN ("article", "course"...)
```

Reply:
it is not a bad thing though, let's merge